File: LanguAI.Backend/LanguAI.Backend/ViewModels/Friendship/UserDiscoveryViewModel.cs
using LanguAI.Backend.Core.Enums;
using LanguAI.Backend.ViewModels.Image;

namespace LanguAI.Backend.ViewModels.Friendship;

public class UserDiscoveryViewModel
{
    public int UserId { get; set; }
    public string Username { get; set; }
    public ImageViewModel ProfilePicture { get; set; }
    public FriendshipStatusEnum? FriendshipStatusEnum { get; set; }
}

File: LanguAI.Frontend/src/api/models/user-discovery-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { FriendshipStatusEnum } from '../models/friendship-status-enum';
import { ImageViewModel } from '../models/image-view-model';
export interface UserDiscoveryViewModel {
  friendshipStatusEnum?: FriendshipStatusEnum;
  profilePicture?: ImageViewModel;
  userId?: number;
  username?: string | null;
}
